lib/components/Keyboard/KeyboardAccessoryView/keyboardAccessoryView.api.json

```diff
@@ -2,6 +2,7 @@
   "name": "KeyboardAccessoryView",
   "category": "infra",
   "description": "View that allows replacing the default keyboard with other components",
+  "note": "This view is useful only for iOS",
```

Comment:
This is used in Android as well

Reply:
This is `KeyboardAccessoryView` it is not only for iOS.

Reply:
fixed